test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -772,14 +697,11 @@ func TestPostFilterPlugin(t *testing.T) {
 				}}})
 
 			// Create the API server and the scheduler with the test plugin set.
```

Comment:
nit: We are not creating API server everytime, right?

Reply:
Yes, updated and applied to all. Thanks.